apps/wallet-mobile/src/features/Swap/common/helpers.ts

```diff
@@ -98,3 +98,29 @@ export const parseOrderTxMetadata = (metadataJson: string): OrderTxMetadata | nu
   const parsedMetadata = parseSafe(metadataJson)
   return isOrderTxMetadata(parsedMetadata) ? parsedMetadata : null
 }
+
+export const formatCustomDate = (dateString: string, locale: string): string => {
```

Comment:
hey @SorinC6 can you check with Product, this will break the i18n and make it always the same format, all other dates now follow the language format

Reply:
will check with them, did not find another way to show it exactly like the design shows 😖